xapian-bindings/xapian-headers.i

```diff
@@ -54,6 +54,7 @@
     %ignore NS::CLASS::CLASS(Internal*);
     %ignore NS::CLASS::CLASS(Internal&);
     %ignore NS::CLASS::operator=;
+    %ignore NS::CLASS::CLASS(CLASS &&);
```

Comment:
Without this change, I get the following error.
```
 Warning 509: Overloaded method Xapian::Document::Document(Xapian::Document const &) effectively ignored,
Warning 509: as it is shadowed by Xapian::Document::Document(Xapian::Document &&).
```
I don't know much about SWIG. From what's already there, I added this fix. Please tell whether this is the way.

Reply:
That seems appropriate to me - these objects are reference counted handles, so it's not useful to explicitly wrap ways to create a copy in the target language.

For some reason Tcl needs the copy ctor wrapped.  I'm not sure we bothered to work out why, but it shouldn't need the move ctor since it doesn't currently have it.